modules/ipi-install-establishing-communication-between-subnets.adoc

```diff
@@ -0,0 +1,153 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="establishing-communication-between-subnets_{context}"]
+= Establishing communication between subnets
+
+In a typical {product-title} cluster setup, all nodes, including the control plane and worker nodes, reside in the same network. However, for edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. This often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability. However, the network must be configured properly before installing {product-title} to ensure that the edge subnets containing the remote worker nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too. 
+
+[IMPORTANT]
+====
+All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes using a manifest. See "Configuring network components to run on the control plane" for details. Deploying a cluster with multiple subnets requires using virtual media.
+====
+
+In the following procedure, the cluster spans two subnets:
+
+- The first subnet (`10.0.0.0`) contains the control plane and local worker nodes.
+- The second subnet (`192.168.0.0`) contains the edge worker nodes.
+
+This procedure details the network configuration required to allow the remote worker nodes in the second subnet to communicate effectively with the control plane nodes in the first subnet and to allow the control plane nodes in the first subnet to communicate effectively with the remote worker nodes in the second subnet.
+
+.Procedure
+
+. Configure the first subnet to communicate with the second subnet.
+
+.. Log in as `root` to a control plane node:
```

Comment:
Review repo guidelines for command step: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output
Ex:
```suggestion
.. Log in as `root` to a control plane node by running the following command:
```